### PR TITLE
Update xml-config-file.md

### PR DESCRIPTION
**Pull request by kdspiess**
@dstrome  - Will likely need some assistance in the proper formatting of the XML table.  We added the following XML values to the reference XML and to the table.

<AutoExitMeetingEnabled>   &    <ZoomMeetingsEnabled>

AutoExitMeetingEnabled  &  ZoomMeetingsEnabled
Thanks!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/xml-config-file.md [hunk 1]
```diff
@@ -106,16 +108,18 @@ If a variable value is of the wrong type, elements are out of order, elements ar
...
+| \<ZoomMeetingsEnabled\>                     | Boolean &#x2777;            | First &#x2776; | Disabled by default. <br/> <br/> If true, enabled direct guest join experience for Zoom meetings.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (JohanFreelancer9) [inline Teams/rooms/xml-config-file.md:111]:
```suggestion
| \<ZoomMeetingsEnabled\>                     | Boolean &#x2777;            | First &#x2776; | Disabled by default. <br/> <br/> If true, enables direct guest join experience for Zoom meetings.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

File: Teams/rooms/xml-config-file.md [hunk 2]
```diff
@@ -106,16 +108,18 @@ If a variable value is of the wrong type, elements are out of order, elements ar
...
+| \<DisableTeamsAudioSharing\>                  | Boolean &#x2777;            | First &#x2776; | Set to true to disable HDMI audio sharing to meeting participants in Teams meeting. The default is false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (JohanFreelancer9) [inline Teams/rooms/xml-config-file.md:120]:
```suggestion
| \<DisableTeamsAudioSharing\>                  | Boolean &#x2777;            | First &#x2776; | Set to true to disable HDMI audio sharing to meeting participants in Teams meetings. The default is false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

File: Teams/rooms/xml-config-file.md [hunk 3]
```diff
@@ -106,16 +108,18 @@ If a variable value is of the wrong type, elements are out of order, elements ar
...
 | \<UseCustomInfoForThirdPartyMeetings\>      | Boolean &#x2777;            | First &#x2776; | Disabled by default and uses conference room account info to join third party meetings. <br/> <br/> If this value is set to true, you must specify both \<CustomDisplayNameForThirdPartyMeetings\>, \<CustomDisplayEmailForThirdPartyMeetings\> must be specified.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (JohanFreelancer9) [inline Teams/rooms/xml-config-file.md:112]:
```suggestion
| \<UseCustomInfoForThirdPartyMeetings\>      | Boolean &#x2777;            | First &#x2776; | Disabled by default and uses conference room account info to join third party meetings. <br/> <br/> If this value is set to true, both \<CustomDisplayNameForThirdPartyMeetings\> and \<CustomDisplayEmailForThirdPartyMeetings\> must be specified.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**